drivers/storage/isilon/storage/isilon_storage.go

```diff
@@ -349,9 +350,6 @@ func (d *driver) VolumeAttach(
 	if err != nil {
 		return nil, "", err
 	}
-	if vol == nil {
-		return nil, "", goof.New("no volumes returned")
-	}
```

Comment:
This code isn't necessary because `d.VolumeInspect` will already return an error if vol is nil. So this is dead code.